.gitignore

```diff
@@ -64,3 +64,4 @@ ssl/
 out/
 .idea_modules/
 
+sendgrid.env
```

Comment:
I don't see this file anywhere? Remove it? or any specific reason you put it?

Reply:
@galipremsagar  actually when I was setting the Environment then I exported it to sendgrid.env and then ran command `source ./sendgrid.env`

Reply:
```suggestion
```
Ah, okay understood. Then we can remove this as we don't have it in our codebase.

Reply:
@galipremsagar done